autode/transition_states/transition_state.py

```diff
@@ -153,26 +168,31 @@ def find_lowest_energy_ts_conformer(self):
         hmethod = get_hmethod() if Config.hmethod_conformers else None
         self.find_lowest_energy_conformer(hmethod=hmethod)
 
-        if len(self.conformers) == 1:
-            logger.warning('Only found a single conformer. '
-                           'Not rerunning TS optimisation')
-            self.set_atoms(atoms=atoms)
-            self.energy = energy
-            self.optts_calc = calc
-            return None
+        # Remove similar TS conformer that are similar to this TS based on root
+        # mean squared differences in their structures
+        thresh = Config.rmsd_threshold if rmsd_threshold is None else rmsd_threshold
+        self.conformers = [conf for conf in self.conformers if
+                           calc_heavy_atom_rmsd(conf.atoms, atoms) > thresh]
 
-        self.optimise(name_ext='optts_conf')
+        logger.info(f'Generated {len(self.conformers)} unique (RMSD < '
+                    f'{thresh} Å) TS conformer(s)')
 
-        if self.is_true_ts() and self.energy < energy:
-            logger.info('Conformer search successful')
+        # Optimise the lowest energy conformer to a transition state
+        if len(self.conformers) > 1:
```

Comment:
What if it finds one TS conf at a lower energy than the original?

Reply:
Fair point, I had assumed the conformer generation would always re-find the same conformer

Reply:
this now needs to do an front-side SN2 TS optimisation for the multistep example..
<img width="1575" alt="sn2_confs" src="https://user-images.githubusercontent.com/39765193/94292836-60d38780-ff55-11ea-98b0-4ffa58fadffd.png">

Reply:
just unlucky – reran and all is well 👌